doc/documentation_contributors.md

```diff
@@ -0,0 +1,116 @@
+Guideline for contributing documentation
+========================================
+The DLite documentation is written in Markdown.
+This include both the README files and documentation found in the `doc/` subdirectory.
+
+Common to both is that the text should be as easy and natural as possible to read from the terminal.
+Hence, the following recommendations should be followed:
+
+* Write one sentence per line, in order to get an easier to read output from `git diff`.
+
+* For README files, use the underline style format for main and sub-headers.
+  For example, start a new page with
+
+      Overview
+      ========
+
+  instead of
+
+      # Overview
+
+  For documentation the style is up to the writer.
+
+* Add two newlines before headers to make them easier to recognise.
+
+  This is especially useful for levels 3-headers and below (which are lines starting with 3 or more hashes (`#`)) to make them easier to recognise as sub-headers.
+
```

Comment:
```suggestion
```
[MD022](https://github.com/DavidAnson/markdownlint/blob/main/doc/md022.md).
Single surrounding lines are fine. Multiple blank lines is not good practice [MD012](https://github.com/DavidAnson/markdownlint/blob/main/doc/md012.md).

Reply:
Removed the bullet point